packages/terra-form-input/tests/jest/InputField.test.jsx

```diff
@@ -9,6 +9,12 @@ it('should render a default InputField component', () => {
   expect(wrapper).toMatchSnapshot();
 });
 
+it('should render a disabled InputField component', () => {
+  const textarea = <InputField disabled inputId="test-input" label="Label" />;
+  const wrapper = shallow(textarea);
+  expect(wrapper).toMatchSnapshot();
+});
+
 it('should render a InputField with props', () => {
```

Comment:
We need to add another jest test to verify the `inputAttrs.disabled` also renders a disabled input field